client/layout/index.jsx

```diff
@@ -243,7 +245,7 @@ class Layout extends Component {
 			};
 		};
 
-		const loadInlineHelp = this.shouldLoadInlineHelp() && ! this.props.disableFAB;
+		const loadInlineHelp = this.shouldLoadInlineHelp() && ! this.props.userAllowedToHelpCenter;
```

Comment:
@escapemanuele I think we should change this check
`inline-help` is false and if the user doesn't satisfy the criteria `10 % 100` he won't see FAB and the hc
```suggestion
		const loadInlineHelp = this.shouldLoadInlineHelp() || ! this.props.userAllowedToHelpCenter;
```

Reply:
updated the suggestion I made a mistake..

Reply:
That makes sense, thanks for the catch! I don't think we can remove `this.shouldLoadInlineHelp()` from the equation, as it won't take in account all the things there. I'll look into this 👍

Reply:
Thinking more about this, I think it makes sense to leave this as it is. If `inline-help` were false, the user wouldn't have seen the FAB before as well.